src/pages/Player.vue

```diff
@@ -1,327 +1,59 @@
-import EventAPIService from "@/services/API/Event.js";
-import VideoFunctionalService from "@/services/Functional/Video.js";
-import ItemFunctionalService from "@/services/Functional/Item.js";
-import ItemModal from "../components/Player/ItemModal.vue";
+import Plio from "@/pages/Embeds/Plio";
 import IconButton from "@/components/UI/Buttons/IconButton.vue";
-import { useToast } from "vue-toastification";
-import { mapActions, mapGetters } from "vuex";
-
-// difference in seconds between consecutive checks for item pop-up
-var POP_UP_CHECKING_FREQUENCY = 0.5;
-var POP_UP_PRECISION_TIME = POP_UP_CHECKING_FREQUENCY * 1000;
-
-// The time period in which Plyr timeupdate event repeats
-// in seconds
-const PLYR_INTERVAL_TIME = 0.05;
-
-// upload data periodically - period in milliseconds
-const UPLOAD_INTERVAL = 10000;
-var UPLOAD_INTERVAL_TIMEOUT = null;
-
-// supports indexOf for older browsers
-if (!Array.prototype.indexOf) {
-  Array.prototype.indexOf = function (elt /*, from*/) {
-    var len = this.length >>> 0;
-
-    var from = Number(arguments[1]) || 0;
-    from = from < 0 ? Math.ceil(from) : Math.floor(from);
-    if (from < 0) from += len;
-
-    for (; from < len; from++) {
-      if (from in this && this[from] === elt) return from;
-    }
-    return -1;
-  };
-}
-
-// supports Array.prototype.fill for older browsers
-if (!Array.prototype.fill) {
-  Object.defineProperty(Array.prototype, "fill", {
-    value: function (value) {
-      // Steps 1-2.
-      if (this == null) {
-        throw new TypeError("this is null or not defined");
-      }
-
-      var O = Object(this);
-
-      // Steps 3-5.
-      var len = O.length >>> 0;
-
-      // Steps 6-7.
-      var start = arguments[1];
-      var relativeStart = start >> 0;
-
-      // Step 8.
-      var k =
-        relativeStart < 0
-          ? Math.max(len + relativeStart, 0)
-          : Math.min(relativeStart, len);
-
-      // Steps 9-10.
-      var end = arguments[2];
-      var relativeEnd = end === undefined ? len : end >> 0;
-
-      // Step 11.
-      var finalValue =
-        relativeEnd < 0 ? Math.max(len + relativeEnd, 0) : Math.min(relativeEnd, len);
-
-      // Step 12.
-      while (k < finalValue) {
-        O[k] = value;
-        k++;
-      }
-
-      // Step 13.
-      return O;
-    },
-  });
-}
 
 export default {
   components: {
-    VideoPlayer,
-    VideoSkeleton,
-    ItemModal,
+    Plio,
     IconButton,
   },
   data() {
     return {
-      plyrConfig: {
-        controls: [
-          "play",
-          "play-large",
-          "progress",
-          "current-time",
-          "mute",
-          "volume",
-          "fullscreen",
-        ],
-
-        ratio: "16:7",
-
-        keyboard: {
-          focused: false,
-          global: false,
-        },
-
-        invertTime: false,
-      },
-      videoId: "", // video Id for the Plio
       source: "unknown", // source from where the plio was accessed - can be passed as param in the plio url
```

Comment:
kept this as we are not anything with source as of now - it is not being sent to the server too.